prody/proteins/interactions.py

```diff
@@ -918,29 +957,52 @@ def calcSaltBridgesDCD(atoms, trajectory, distA=4.5):
         except TypeError:
             raise TypeError('coords must be an object '
                             'with `getCoords` method')
+    SBs_all = []
 
-    if isinstance(trajectory, Atomic):
-        trajectory = Ensemble(trajectory)        
+    if trajectory is not None:
+        if isinstance(trajectory, Atomic):
```

Comment:
you could have an elif here that if it's not Trajectory or Ensemble then raise a type error